prereise/cli/tests/test_helpers.py

```diff
@@ -0,0 +1,56 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from prereise.cli.helpers import (
+    add_data_source_to_download_parser,
+    validate_date,
+    validate_file_path,
+)
+
+
+@pytest.fixture
+def data_source():
+    data_source_mock = MagicMock()
+    data_source_mock.command_name = "command_name"
+    data_source_mock.command_help = "command_help"
+    data_source_mock.extract_arguments = [
+        {"command_flags": ["--command"], "keyarg": "value"}
+    ]
+    data_source_mock.extract = MagicMock()
+    return data_source_mock
+
+
+def test_validate_date():
+    validate_date("2020-01-03")
```

Comment:
For completeness, we could also assert that it returns the value passed in. Similarly for `test_validate_file_path`

Reply:
Thanks! Fixed